panda/src/navigation/config_navigation.cxx

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
All your source/header files (not the Recast/Detour ones) need to have a file header with the license, author and date, like in other Panda source files.

Reply:
Done

Reply:
Still missing from config_navigation.*